website/docs/cert-manager.md

```diff
@@ -0,0 +1,134 @@
+---
+id: cert-manager
+title: Injecting Certificates from Cert Manager
```

Comment:
We should make this more generic: "Managing certificates outside of Gatekeeper"

Reply:
Or similar, not in love with that title.

Reply:
I understand the concern. 
However, I use here the `ca-injector` from cert-manager to change the CABundle in the webhook (through annotations). It is actually this precise functionality I am more interested in. 
Without this CAInjector, it would require the controller code to manage this injection (through the CA contained in the certificate passed in parameter) or to make the `caBundle` a helm parameter. This second option must be a simpler implementation.